src/sql/magic.py

```diff
@@ -574,7 +574,8 @@ def interactive_execute_wrapper(**kwargs):
                 conn,
                 command.sql,
                 self,
-                parameters=user_ns if self.named_parameters else None,
+                parameters=user_ns if self.named_parameters == "true" else None,
+                enable_named_params=self.named_parameters,
```

Comment:
Do we really need the `enable_named_params` or we can modify `parameters` ? Like maybe set it to {} if `disable`. Not necessarily `{}` but you can think through it and decide the best approach

Reply:
Thank you for the tip. I removed `enable_named_params` and modified parameters [here](https://github.com/ploomber/jupysql/pull/980/files#diff-527caf90dd08297f5253e826bd98fea0b6f4fbd44fe9f20472976f77868dac06R573-R585). Now, parameters = `None` if named parameters are `disabled`, `{}` if named parameters are `false`, and `user_ns` if named parameters are `true`. These changes are also reflected in [`connection.raw_execute()`](https://github.com/ploomber/jupysql/pull/980/files#diff-62e3d6289e3242b271e6df3a7f201ab003f8ce73a5daf1a824cc2b4d892513d1R773-R783).